pretty.c

```diff
@@ -98,6 +98,16 @@ static void print_device(struct json_object *obj)
 		uint16_t device = get_object_object_uint64(data_obj, "device");
 		printf(" %04x:%04x", vendor, device);
 		break;
+#if LIBDRM_VERSION >= VERSION(2, 4, 75)
```

Comment:
You should add `DRM_BUS_PLATFORM` as a define in `compat.h` instead, since there isn't any use of DRM types or functions here.

Just trying to keep those #ifs in the source code to a minimum.

Reply:
Indeed, that's better. Plus the defines were already there.